docs/index.md

```diff
@@ -4,6 +4,14 @@ title: Home
 
 # ![Ragna logo](assets/brand/logo-lockup-horizontal/logo-lockup-horizontal.png)
 
+- Here is a nested list with a tab width of 2
+  - First item
+  - Second item
+- Here is a nested list with a tab width of 4
+    - First item
+    - Second item
```

Comment:
This is just here to showcase a tab width 2 vs 4. 15bb981ca271895f94adef0d16c3034e539382a3 needs to be reverted before merge.

Reply:
Here is the rendered result

![image](https://github.com/Quansight/ragna/assets/6849766/f2ea4a9a-9cf7-43d3-9134-c90ee80f04bb)

Using only 2 tabs flattens the sublist.